app/src/main/java/org/oppia/app/profileprogress/ProfilePictureActivityViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.app.profileprogress
+
+import android.net.Uri
+import androidx.databinding.ObservableField
+import org.oppia.app.model.ProfileAvatar
+import org.oppia.app.viewmodel.ObservableViewModel
+
+/** [ViewModel] for displaying User image in Activity. */
```

Comment:
Nit user should small caps: `/** [ViewModel] for displaying user image in Activity. */`